api/src/pcapi/routes/external/users_subscription.py

```diff
@@ -105,13 +105,32 @@ def dms_webhook_update_application_status(form: dms_validation.DMSWebhookRequest
             status=import_status,
         )
 
-    if form.state == api_demarches_simplifiees.GraphQLApplicationStates.accepted:
-        subscription_api.start_workflow(user, str(form.dossier_id), application)
-
-    elif form.state == api_demarches_simplifiees.GraphQLApplicationStates.draft:
+    if form.state == api_demarches_simplifiees.GraphQLApplicationStates.draft:
         subscription_messages.on_dms_application_received(user)
+        subscription_api.start_workflow(user, str(form.dossier_id), application)
+        logger.info(
+            "DMS Application %d created.",
```

Comment:
Je proposerais bien de mettre l'identifiant à la fin du message : ça facilite la recherche de la chaîne "DMS application created" (dans les logs ou dans le code) :  

```suggestion
            "DMS application created: %s",
```

Idem pour le log de refus plus bas.

(On pourrait même omettre l'identifiant dans le message, puisqu'il est loggé dans `extra`.)